package.json

```diff
@@ -50,6 +48,8 @@
     "unibeautify": "0.17.0"
   },
   "dependencies": {
-    "read-pkg-up": "^4.0.0"
+    "read-pkg-up": "^4.0.0",
+    "cosmiconfig": "5.0.6",
+    "csscomb": "4.2.0"
```

Comment:
`csscomb` should be a `devDepenency` and `peerDependency`, which it already is: https://github.com/Unibeautify/beautifier-csscomb/blob/master/package.json#L34-L42

Reply:
@muuvmuuv I think the rest looks correct, however, `csscomb` should be under `peerDependencies` and `devDependencies` and NOT `dependencies`.